### PR TITLE
refactor(console): more specific links in guide further readings section

### PR DESCRIPTION
**Pull request by charIeszhao**
## Summary

Update "Further readings" section in each Console integration guide. Replace "Full guide" link with two more specific links:
- Get user information
- API resources

## Testing

image

## Checklist

~~- [ ] `.changeset`~~
~~- [ ] unit tests~~
~~- [ ] integration tests~~
~~- [ ] necessary TSDoc comments~~

### CODE REVIEW

**Comment by xiaoyijun** [comment]
Does every guide include these two sections?

**Review by wangsijie — APPROVED** [review]

**Comment by charIeszhao** [comment]
> Does every guide include these two sections?

This needs to be double checked by all devs

**Review by gao-sun — APPROVED** [review]

**Review by darcyYe — APPROVED** [review]

File: packages/console/src/mdx-components/Steps/FurtherReadings.tsx [hunk 1]
```diff
@@ -15,11 +15,18 @@ function FurtherReadings(props: Props, ref?: Ref<HTMLDivElement>) {
...
       <ul>
         {fullGuide && (
-          <li>
-            <TextLink href={fullGuide.url} targetBlank="noopener">
-              {fullGuide.title}
-            </TextLink>
-          </li>
+          <>
+            <li>
+              <TextLink href={`${fullGuide.url}/#get-user-information`} targetBlank="noopener">
```
Comment (gao-sun) [inline packages/console/src/mdx-components/Steps/FurtherReadings.tsx:20]: some guides does not have these two sections

Comment (charIeszhao) [reply packages/console/src/mdx-components/Steps/FurtherReadings.tsx:20]: I'll take a look and have it displayed conditionally

**Pull request closed without merging**